javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/resolution/typesolvers/CombinedTypeSolver.java

```diff
@@ -34,13 +35,29 @@
 
     private TypeSolver parent;
     private List<TypeSolver> elements = new ArrayList<>();
+    private Predicate<Exception> errorFilter;
 
     public CombinedTypeSolver(TypeSolver... elements) {
+        this(ExceptionFilters.IGNORE_ALL, elements);
```

Comment:
IGNORE_ALL should not be the default

Reply:
Whooops that's due to a refactor, will fix.